channels/us.m3u

```diff
@@ -786,7 +786,7 @@ https://playout4multirtmp.tulix.tv/live8/Stream1/playlist.m3u8
 https://granicusliveus4-a.akamaihd.net/cerritos/G0010_002/playlist.m3u8
 #EXTINF:-1 tvg-id="CGTV.us" tvg-name="CGTV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/f6dfLXA.png" group-title="Legislative",CGTV
 https://reflect-live-champaign.cablecast.tv/live/CELL-296k-234p/CELL-296k-234p.m3u8
-#EXTINF:-1 tvg-id="Channel5.us" tvg-name="Channel 5 (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/sCRPkPk.jpg" group-title="Series",Channel 5 (Adult Swim 24/7 Stream) [Geo-blocked]
+#EXTINF:-1 tvg-id="Channel5.us" tvg-name="Channel 5 (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/sCRPkPk.jpg" group-title="Animation",Channel 5 (Adult Swim 24/7 Stream) [Geo-blocked]
```

Comment:
Adult Swim as a channel itself shows variety of content. Entertainment was right category